src/routes/_homepage/_search.js

```diff
@@ -1,13 +1,15 @@
-export function getQuery(category, subcategory, cityCode, cityLabel) {
+export function getQuery(category, subcategory, cityCode, cityLabel, radius) {
   const parameters = {
     cat: category,
     sub: subcategory,
     city: cityCode,
     cl: cityLabel,
+    ra: radius,
```

Comment:
```suggestion
    radius,
```

Attention, l'API attend `radius`, pas `ra`.
(Ce qui est un peu crad, c'est que j'utilisais la même méthode pour préparer les parametres de l'appel back que pour la page en front.)